.circleci/config.yml

```
@@ -565,6 +565,14 @@ jobs:
         description: Number of fuzz runs to apply
         type: integer
         default: 512
+      test_invariant_runs:
+        description: Number of invariant runs to apply
+        type: integer
+        default: 64
+      test_invariant_depth:
+        description: Depth of invariant runs
+        type: integer
+        default: 32
```

Lets swap these, since having a larger depth matters a lot
```suggestion
      test_invariant_runs:
        description: Number of invariant runs to apply
        type: integer
        default: 32
      test_invariant_depth:
        description: Depth of invariant runs
        type: integer
        default: 64
```